packages/node_modules/brightspace-auth-keys-dynamodb-store/src/public-key-table-setup.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+function createPublicKeyTable(db, tablename, readCapacity, writeCapacity) {
+	var params = {
+		AttributeDefinitions: [
+			{
+				AttributeName: 'PublicKey',
+				AttributeType: 'S'
+			}
+		],
+		KeySchema: [
+			{
+				AttributeName: 'PublicKey',
+				KeyType: 'HASH'
+			}
+		],
+		ProvisionedThroughput: {
+			ReadCapacityUnits: readCapacity || 5,
+			WriteCapacityUnits: writeCapacity || 5
+		},
+		TableName: tablename
+	};
+
+	return db.createTable(params).promise().then(() => {
+		return setTtl(db, tablename);
+	}).catch((e) => {
+		// ResourceInUseException indicates the table already exists
+		if (e.code !== 'ResourceInUseException') {
+			throw e;
+		}
+	});
+}
+
+function setTtl(db, tablename) {
+	var params = {
+		TableName: tablename,
+		TimeToLiveSpecification: {

```

Comment:
I can't find the dynamo documentation that actually describes what the value of the TTL column is/interpreted as...?

Reply:
It shows up in the popup in the console "TTL is a mechanism to set a specific timestamp for expiring items from your table. The timestamp should be expressed as an attribute on the items in the table. The attribute should be a Number data type containing time in epoch format. Once the timestamp expires, the corresponding item is deleted from the table in the background."